crates/miden-objects/src/account/component/template/storage/init_storage_data.rs

```diff
@@ -34,3 +41,93 @@ impl InitStorageData {
         self.storage_placeholders.get(key)
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Should this test actually go into the `toml` module? Originally, I was trying to find them there since that's where the implementation of the function they are testing is in.

Also, these tests should be done only in the `std` context, right?

Reply:
Moved this to the `toml` module (which is based on `std`)